src/ui/component/tag/tag.tsx

```diff
@@ -8,7 +8,7 @@ import classNames from 'classnames'
 type TagProps = {
   tagName: string
   classes?: { main?: string; name?: string; icon?: string }
-  onDelete?: () => void
+  onDelete?: (tagName: string) => void
```

Comment:
I don't believe it's necessary, as capturing the context can be achieved through the callback function's closure. The pattern to use is: a context is passed to the callback function only if it's essential to understand the interaction or to clarify any ambiguity from the caller perspective. It's not required in this case. Otherwise, for consistency, we would need to apply this pattern to the callbacks of all other components.

Reply:
Oh yes! Thanks for the detailed explanation, value is already provided by the parent's context. Updated in baf4a949.